src/arithmetic.jl

```diff
@@ -382,7 +384,7 @@ end
 for T in (:HomogeneousPolynomial, :TaylorN)
     @eval function /(b::$T{Taylor1{S}}, a::Taylor1{T}) where
             {T<:NumberNotSeries,S<:NumberNotSeries}
-        @inbounds aux = b[1] / a
+        @inbounds aux = b.coeffs[1] / a
```

Comment:
`b[0]`?

Reply:
Same as above; `b` may be an HomogeneousPolynomial...